apps/taxes/src/pages/api/register.ts

```diff
@@ -25,4 +37,46 @@ export default createAppRegisterHandler({
       return true;
     },
   ],
+  /**
+   * TODO Unify with all apps - shared code. Consider moving to app-sdk
+   */
+  async onRequestVerified(req, { authData: { token, saleorApiUrl }, respondWithError }) {
+    const logger = createLogger({
+      context: "onRequestVerified",
+    });
+
+    try {
+      const client = createClient(saleorApiUrl, async () => {
+        return {
+          token,
+        };
+      });
+
+      const saleorVersion = await client
+        .query<SaleorVersionQuery>(SaleorVersion, {})
+        .toPromise()
+        .then((res) => {
+          return res.data?.shop.version;
```

Comment:
Please check if running Saleor core from the main branch will pass this test. AFAIR they do not have version on main, just use `dev` tag.
If so, should we add edge case for it?

Reply:
@krzysztofwolski main is on master-staging right? Apps install there properly AFAIK

Reply:
Great!